src/check/package-json.test.js

```diff
@@ -28,7 +40,19 @@ tap.ok(
   'package.json has "author" property'
 );
 
-tap.equal(pkg.main, 'es5/plugin.js', 'package.json "main" is "es5/plugin.js"');
+tap.equal(pkg.main, 'dist/es5/index.js', 'package.json "main" is "dist/es5/index.js"');
+tap.equal(pkg['jsnext:main'], 'src/js/index.js', 'package.json "main" is "src/js/index.js"');
+
+tap.ok(
+  isNonEmptyString(pkg['generator-videojs-plugin'].version),
+  'package.json "generator-videojs-plugin.version" exists'
+);
+
+tap.ok(_.isPlainObject(pkg.spellbook), 'package.json has "spellbook" object');
+
+Object.keys(pkg.spellbook).forEach(k => {
+  tap.equal(typeof pkg.spellbook[k], 'boolean', `package.json "spellbook.${k}" is a boolean`);
```

Comment:
How's that? It only looks like they're ever [set to booleans](https://github.com/videojs/generator-videojs-plugin/blob/master/src/app/package-json.js#L130-L139)...